backend/services/routes/categories.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const router = express.Router();
+
+const axios = require("axios");
+axios.defaults.baseURL = "http://localhost:3000";
+
+router.get("/", (req, res, next) => {
+  axios
+    .get(`/api/categories`)
+    .then((response) => {
+      res.render("categories", { categories: response.data });
+    })
+    .catch((error) => {
+      next(error);
+    });
+});
+
+router.get("/:category", (req, res, next) => {
+  const { category } = req.params;
+
+  axios
```

Comment:
Why do we have to call on another endpoint inside the actual endpoint? Just use core layer methods.